piws/importer/base.py

```diff
@@ -358,8 +381,14 @@ def _create_assessment(self, assessment_struct, subject_eids, study_eid,
                 entity_name="Assessment",
                 **assessment_struct)
             assessment_eid = assessment_entity.eid
-            self.already_related_subjects[assessment_eid] = [
-                e.eid for e in assessment_entity.subjects]
+            self.already_related_subjects[assessment_eid] = []
+            if not is_created:
+                rql = ("Any S Where A is Assessment, A eid {}, "
+                       "A subjects S".format(assessment_eid))
+                rset = self.session.execute(rql)
```

Comment:
self.already_related_subjects[assessment_eid] = [row[0] for row in self.session.execute(rql)]